Projects/Server.Tests/Utility/ActivatorUtilTests.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using Server.Utilities;
+using Xunit;
+
+namespace Server.Tests
+{
+    public class TestZeroParamsClass
+    {
+    }
+
+    public class TestTwoParamsClass
+    {
+        public int Amount;
+        public string Name;
+
+        public TestTwoParamsClass(int amount, string name)
+        {
+            Amount = amount;
+            Name = name;
+        }
+    }
```

Comment:
Not sure what your policy on this is, but i try to put classes, created for the purpose of unit tests, inside these classes as private ones, otherwise 1 file per class policy should be adhered.